Network/DNS/StateBinary.hs

```diff
@@ -125,7 +125,7 @@ get16 = ST.lift getWord16be <* addPosition 2
     getWord16be = do
         a <- word8'
         b <- word8'
-        return $ a * 256 + b
+        return $ a * 0x100 + b
```

Comment:
Yes, most coders know that `256` is `0x100 but few coders can tell the decimal value of `0x1000000`. Changing this one so its in the same form as the others below.